### PR TITLE
resolve #432 remove AASd-120: idShort also allowed for elements

### LINKED ISSUE #432: Relax Constraint AASD-120 for SMT allow idShort in SML
**Is your feature request related to a problem? Please describe.**
For Submodel Template creation it is helpful to have an idShort for the element type within the SML.
This is mainly the case for SML of SMC or SML because there are separate tables for SMC and SML. For properties it is more difficult. 
See discussion in https://github.com/admin-shell-io/submodel-templates/issues/65

`Constraint AASd-120: idShort of submodel elements being a direct child of a SubmodelElementList shall not be specified.`

could be relaxed to

`Constraint AASd-120: idShort of submodel elements being a direct child of a SubmodelElementList shall not be specified within Submodels with kind=Instance.`

**Describe alternatives you've considered**

- do not relax
-- instead create ConceptDescriptions with idShort of the corresponding elements

- remove completely, i.e. also allow idShort in Submodels with kind=Instance for better readability

AASd-120 was a further restriction of
`Constraint AASd-117: idShort of non-identifiable Referables not being a direct child of a SubmodelElementList shall be specified.`

### PR DESCRIPTION
**Pull request by BirgitBoss**
#432 remove AASd-120: idShort also allowed for elements within a SubmodelElementList

### CODE REVIEW

**Comment by BirgitBoss** [comment]
see https://github.com/admin-shell-io/aas-specs-api/issues/334 for impact on API: not yet resolved

File: documentation/IDTA-01001/modules/ROOT/pages/includes/idshortpath.adoc [hunk 1]
```diff
@@ -12,9 +12,12 @@ Plattform Industrie 4.0; Anna Salari, Publik. Agentur für Kommunikation GmbH, d
...
 === Format "Path" (IdShortPath Serialization) in JSON
 
-To get only the idShort paths of a submodel element hierarchy, the serialization format is specified in terms of an idShortPath notation to be returned in an JSON array. 
-The notation differs depending on whether a SubmodelElementCollection or a SubmodelElementList is present. In the first case, the submodel element's idShort 
-is separated by "." (dot) from top level down to child level. In the second case, square brackets with an index "[<Index>]" are appended after the idShort of the containing SubmodelElementList.
+To get only the idShort paths of a submodel element hierarchy, the serialization format is specified in terms 
+of an idShortPath notation to be returned in an JSON array. 
+For SubmodelElementLists the elements can be identified via its index or via its idShort.
+For identification via idShort the submodel element's idShort 
+is separated by "." (dot) from top level down to child level.
```
Comment (BirgitBoss) [inline documentation/IDTA-01001/modules/ROOT/pages/includes/idshortpath.adoc:19]: there needs to be a final decision on handling with "." notation or via brackets

**Comment by BirgitBoss** [comment]
see https://github.com/admin-shell-io/aas-specs/issues/432#issuecomment-2476128451. Also References need to be considered

**Review by s-heppner — APPROVED** [review]

**Comment by BirgitBoss** [comment]
use  no idShort in IdShortPath for SML #501  instead

**Pull request closed without merging**